examples/scripts/comparison_examples/exp_UKF.py

```diff
@@ -28,17 +28,17 @@
 t_eval = np.linspace(0, 20, 10)
 true_inputs = parameters.as_dict("true")
 values = model.predict(t_eval=t_eval)
-values = values["2y"].data
+values = values["y_0"].data
```

Comment:
It's a bit confusing to see `y0` as the initial state, and `y_0` as the first state vector. Since the model works with array outputs `y`, it would be helpful to show how this would work in the example (e.g. for 2 states). Otherwise, we can simplify `y_0` to `y`.